tutorial2_quantities_and_amounts/demo/main.js

```diff
@@ -41,45 +33,30 @@ d3.csv("../../data/surveyResults.csv", d => {
     .attr("width", width)
     .attr("height", height);
 
-  // main group
-  const group = svg
-    .selectAll("g")
+  // append rects
+  const rect = svg
+    .selectAll("rect")
     .data(data)
-    .join("g")
-    .attr(
-      "transform",
-      (d, i) => `translate(${xScale(i)}, ${height - margin.bottom})`
-    );
-
-  // rects
-  const rects = group
-    .append("rect")
+    .join("rect")
+    .attr("y", d => yScale(d.count))
+    .attr("x", d => xScale(d.activity))
     .attr("width", xScale.bandwidth())
-    .attr("height", 0);
+    .attr("height", d => height - margin.bottom - yScale(d.count));
 
-  // text
-  group
-    .append("text")
-    .attr("class", "hover-text")
-    .attr("dy", "1.25em")
-    .attr("x", xScale.bandwidth() / 2)
-    .text(d => d.d3);
+  // append text
+  const text = svg
+    .selectAll("text")
+    .data(data)
+    .join("text")
+    .attr("class", "label")
+    .attr("x", d => xScale(d.activity) + (xScale.bandwidth() / 2))
```

Comment:
I feel like a comment might be helpful here for why we are doing this -- not a blocker though